pandas/core/dtypes/concat.py

```diff
@@ -97,7 +97,7 @@ def _cast_to_common_type(arr: ArrayLike, dtype: DtypeObj) -> ArrayLike:
     return arr.astype(dtype, copy=False)
 
 
-def concat_compat(to_concat, axis: int = 0):
+def concat_compat(to_concat, axis: int = 0, pretend_axis1: bool = False):
```

Comment:
why don't you just allow axis=None and call it that case. this is very odd naming here (appreciate the de-duplication that this allows); so i guess a better question is this temporary?

Reply:
> why don't you just allow axis=None and call it that case. this is very odd naming here

I want the naming to be very clear that this is a dont-try-this-at-home kludge (need to update the docstring to that effect)

> so i guess a better question is this temporary?

inasmuch as it wont be needed once we have 2D EAs, yes.

Reply:
ok to the extent this is not removed anytime soon, can you come up with a better argument name or another way of doing this?

Reply:
renamed + green